app/models/key.rb

```diff
@@ -24,10 +24,12 @@ def shell_id
 
   def add_to_authorized_keys
     AuthorizedKeysManager.add(self.id, self.key)
+    system(Ontohub::Application.config.data_root + '/.ssh/cp_keys')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.